### ISSUE TITLE
Pending FoodResources' FoodResourceTypes do not display correctly.

### ISSUE THREAD

**Issue by weronica** [hack4impact/gpcah#47]
**Steps to recreate:**
1. Submit a food resource review using the "Propose Resource" page.
2. Go to the "Manage Food Resources" page (under the "Admin" tab.)
3. Expand the pending resource that you just created. 

**Expected behavior:**
"Resource Type" should list the food resource's type (e.g., "Farmers' Market," "SHARE Host Site," etc.)

**Actual behavior:**
"Resource Type" is blank.

**Comment by weronica** [comment]
Observations:
- When a pending resource is "Approved" by an admin, it shows up correctly in the "Approved Resources" section (even though it had no "resource type" when it was pending.)
- If the below section of code is added to the "Approved Resources" section of the page on `admin_resources.html`, all food resources' types are correctly displayed (even those that were just approved.)

```
<div class="row">
    <div class="small-3 columns">
        Resource Type:
    </div>
    <div class="small-9 columns">
        {{ resource.food_resource_type.name_singular }}
    </div>
</div>
```
- If a pending resource is queried directly from the python database console, then it does have the correct `FoodResourceType` associated with it.
```
>>> fr = FoodResource.query.filter_by(name="testing").first() # testing was a pending FoodResource (i.e., is_approved==False)
>>> fr.food_resource_type
<app.models.FoodResourceType object at 0x1078341d0>
>>> fr.food_resource_type.name_singular
u"Farmers' Market"
```

**Comment by weronica** [comment]
Fixed by @sandlerben in #63.
